setup.cfg

```diff
@@ -38,11 +38,20 @@ exclude =
 [options.extras_require]
 tests = pytest
 
+[bdist]
+formats = rpm
+
+[bdist_rpm]
+requires = python, numpy, python-matplotlib, python-periodictable, PyQt4, mantidnightly
+build_requires = python, python-setuptools, PyQt4, versioningit~=1.1
+doc_files = README.md
```

Comment:
we are building rpm for iMars3D?

Reply:
also, why do we need
- python-periodictable
- PyQt4
- mantidnightly
for iMars3D?

Reply:
We probably dont, this was copied over wholesale when I was creating the file based on Addie, forgot to delete it.